usr/share/rear/conf/default.conf

```diff
@@ -2787,7 +2787,7 @@ BLOCKCLONE_TRY_UNMOUNT="no"
 COPY_AS_IS_BAREOS=( /etc/bareos /var/spool/bareos )
```

Comment:
Here should be also the systemd stuff

Reply:
systemd service file is created dynamically.